docs/guides/manage_serverless_resources.ipynb

```diff
@@ -0,0 +1,334 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transpile_worker_references = [\n",
+    "    transpile_remote(circuit, optimization_level, backend)\n",
+    "    for circuit in circuit_list\n",
+    "]\n",
+    "\n",
+    "results = get(transpile_worker_references)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Exploring different task configurations\n",
+    "\n",
+    "You can flexibly allocate CPU, GPU, and memory for your tasks via `@distribute_task()`. For Qiskit Serverless on IBM Quantum Platform,  each program is equipped with 16 CPU cores and 32 GB RAM, which can be allocated dynamically as needed. \n",
+    "\n",
+    "CPU cores can be allocated with as full CPU cores, or even fractional allocations, as shown below.\n",
+    "\n",
+    "Memory is allocated as number of bytes. Remember, there are 1024 bytes in a kilobyte, 1024 kilobytes in a megabyte, and 1024 megabytes in a gigabyte. To allocate 2 GB of memory for your worker, you need to allocate `\"mem\": 2 * 1024 * 1024 * 1024` "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@distribute_task(target={\n",
+    "    \"cpu\": 16,\n",
+    "    \"mem\": 32 * 1024 * 1024 * 1024\n",
+    "})\n",
+    "def transpile_remote(circuit, optimization_level, backend):\n",
+    "    return None"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Automatic QPU selection\n",
+    "\n",
+    "The example below demonstrates how to use `IBMQPUSelector` to automate the process of selecting which qubits will be used from a set of available QPUs. Instead of manually selecting a QPU, Qiskit Serverless automatically allocates a QPU according to desired criteria. `IBMQPUSelector`s are imported from server-side `qiskit_serverless_tools.selectors` modules, and must be invoked from your uploaded program.\n",
+    "\n",
+    "Here, `IBMLeastNoisyQPUSelector` finds the QPU, among the ones available to you through your IBM Quantum account, that yields the least-noisy qubit subgraph for the input circuit. \n",
+    "\n",
+    "For each `IBMQPUSelector`, the context is set in the constructor. All `IBMQPUSelectors` require Qiskit Runtime credentials. The `IBMLeastNoisyQPUSelector` requires a circuit and transpile options specifying how the circuit should be optimized for each QPU when determining the most optimal QPU and qubit layout. \n",
+    "\n",
+    "All `IBMQPUSelectors` implement a get_backend method, which retrieves the optimal QPU with respect to the given context. The get_backend method also allows for additional filtering of the QPUs. It is implemented using the same interface as the [QiskitRuntimeService.backends method](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#backends)."
```

Comment:
```suggestion
    "This example demonstrates how to use `IBMQPUSelector` to automate the process of selecting which qubits to use from a set of available QPUs. Instead of manually selecting a QPU, Qiskit Serverless automatically allocates a QPU according to desired criteria. `IBMQPUSelector`s are imported from server-side `qiskit_serverless_tools.selectors` modules, and must be invoked from your uploaded program.\n",
    "\n",
    "Here, `IBMLeastNoisyQPUSelector` finds the QPU that yields the least-noisy qubit subgraph for the input circuit, from among the QPUs available to you through your IBM Quantum account. \n",
    "\n",
    "For each `IBMQPUSelector`, the context is set in the constructor. All `IBMQPUSelectors` require Qiskit Runtime credentials. The `IBMLeastNoisyQPUSelector` requires a circuit and transpile options specifying how the circuit should be optimized for each QPU, to determine the most optimal QPU and qubit layout. \n",
    "\n",
    "All `IBMQPUSelectors` implement a `get_backend` method, which retrieves the optimal QPU with respect to the given context. The `get_backend` method also allows for additional filtering of the QPUs. It is implemented using the same interface as the [QiskitRuntimeService.backends method](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#backends)."
```